### PR TITLE
HPCC-15839 Fixed chained stop() issues

### PR DESCRIPTION
**Pull request by jakesmith**
Signed-off-by: Jake Smith jake.smith@lexisnexis.com

### CODE REVIEW

**Comment by jakesmith** [comment]
@richardkchapman - please review

File: thorlcr/activities/piperead/thprslave.cpp [hunk 1]
```diff
@@ -455,7 +463,6 @@ class CPipeThroughSlaveActivity : public CPipeSlaveBase
             throw wrexc.getClear();
         if (retcode!=0 && !(flags & TPFnofail))
             throw MakeActivityException(this, TE_PipeReturnedFailure, "Process returned %d", retcode);
-        PARENT::stop();
```
Comment (ghalliday) [inline thorlcr/activities/piperead/thprslave.cpp]: Should this line still be present?

**Comment by ghalliday** [comment]
@jake one comment

File: thorlcr/activities/piperead/thprslave.cpp (hunk 1 above)
Comment (jakesmith) [inline thorlcr/activities/piperead/thprslave.cpp]: Not really related, but noticed during change it was already called a few lines before (on line 460)

**Pull request merged**